modules/ding_groups/ding_groups.info

```diff
@@ -14,7 +13,6 @@ dependencies[] = list
 dependencies[] = manualcrop
 dependencies[] = media
 dependencies[] = menu_position
-dependencies[] = metatag_panels
```

Comment:
Is this removed on purpose? Does your "new theme" support the metatag changes that @holt83 made?